boards/arm/xiao_ble/xiao_ble_defconfig

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Since the adafruit bootloader is pre-flashed, we should add the build output uf2 config here, to generate that by default.

Reply:
Please see https://github.com/zephyrproject-rtos/zephyr/pull/46358#discussion_r895357919